### ISSUE TITLE
Youtube videos not playing.

### ISSUE THREAD

**Issue by tushar2004** [moay/afterglow#92]
Hi,
Thanks for the awesome package.

But I think there's a little bug. It doesn't play youtube videos. The example video which is there on the site is even not working. Once you click on the video, the loader just keeps on rotating and doesn't do anything.

Thanks.

**Comment by Shivansh2407** [comment]
Can you please explain the issue details so that I can help You out
I would want to work on it Please

**Comment by tushar2004** [comment]
Here you can look up the problem.
**_http://docs.afterglowplayer.com/docs/playing-a-youtube-video_**

**Comment by Shivansh2407** [comment]
Please reserve this issue for me .I would work on it and let you know soon.

**Comment by Edtigoza** [comment]
> Hi,
> Thanks for the awesome package.
> 
> But I think there's a little bug. It doesn't play youtube videos. The example video which is there on the site is even not working. Once you click on the video, the loader just keeps on rotating and video is not played.
> 
> Thanks.

yes, it no work, I give play and no work, in my site no work, it no work  http://docs.afterglowplayer.com/v1.1/docs/playing-a-youtube-video

**Comment by dedpnda** [comment]
I wanted to know if there was any progress on this? I had my website up for a long time not knowing this was not linking people to my work. 

Thank you all for your hard work. I don't know how to code so thanks to everyone involved with this project. All your work is appreciated by all of us. Thank you.

**Comment by palashagrawal03** [comment]
I am facing the same problem as showing in above screenshot
